front/src/common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react';
+import { ModalContext } from './ModalProvider';
+
+type Props = {
+  children: JSX.Element;
+  withCloseButton?: boolean;
+  withBorderBottom?: boolean;
+};
+
+export default function ModalHeaderSNCF({
+  children,
+  withCloseButton = false,
+  withBorderBottom = false,
+}: Props) {
+  const { closeModal } = useContext(ModalContext);
+  return (
+    <>
+      <div className="modal-header">
+        {children}
+        {withCloseButton && (
+          <button type="button" className="close" aria-label="Close" onClick={closeModal}>
+            <span aria-hidden="true">&times;</span>
+          </button>
+        )}
+      </div>
+      {withBorderBottom ? (
+        <div className="modal-header modal-header-border-bottom">
+          <div />
+        </div>
+      ) : null}
```

Comment:
What is it for ? Why is there an empty div ?

Reply:
Without it, there was no display of line as it should be (see scss code)